.github/workflows/main.yaml

```diff
@@ -160,10 +160,21 @@ jobs:
         run: scripts/get-version --github
       - run: ci/create-new-project ${HOME}/workspace geomapfish
       - run: (cd ${HOME}/workspace/geomapfish/; ./build)
+
       - name: Publish
         run: |
           c2cciutils-publish
+          c2cciutils-publish --group=full
+        if: >
+          github.ref != format('refs/heads/{0}', env.MAIN_BRANCH)
+          && github.repository == 'camptocamp/c2cgeoportal'
+      - name: Publish
```

Comment:
Is it really usefull to have two steps with the same name and same conditions (if)

Reply:
The conditions are different on time we have `==` and the other `!=` :-)